test/components/Pagination-test.js

```diff
@@ -0,0 +1,89 @@
+import {test} from 'tape';
+import Pagination from '../../lib/components/Pagination';
+import React from 'react/addons';
+const TestUtils = React.addons.TestUtils;
+
+/**
+ * TODO: move to utils for re-use
+ */
+const createComponent = (component, props, ...children) => {
```

Comment:
Enzyme is quite unstable, true. So bleeding edge. And more then of it depends on mocha too =(